core/src/main/scala/KVGetResult.scala

```diff
@@ -5,7 +5,7 @@ import argonaut._, Argonaut._
 /** Case class representing the response to a KV "Read Key" API call to Consul */
 final case class KVGetResult(
   key:         String,
-  value:       String,
+  value:       Option[String],
```

Comment:
This is going to be a binary breaking change and require another major version, though I can't think of a better way.  It would probably be incorrect and/or evil to conflate the empty string with a null value here.

@aldiyen, do you have any thoughts here?

Reply:
Heh interesting, the Consul documentation doesn't seem to say anything about that. I cannot think of any better alternative either.

I think either `kvGetRaw` or `kvGetJson` needs to change as well to handle the null value in Consul's KV store situation, as I suspect that `kvGetRaw` will return `Some(Array())` and then the JSON parsing will blow up, which seems more confusing than having `kvGetJson` spit out a `None`. I haven't tested this theory though.

Reply:
You are absolutely right. Corrected this case.